### PR TITLE
Exit process

### PR DESCRIPTION
**Pull request by JiejayLan**
### Finish the exit process part5
- [x] print a list the last 4 commands to the terminal
- [x] print a detailed list of all content of the current directory (similar to ls -l). 
- [x] wait for the user to hit the “return” key to terminate the shell and return the control to the original shell program on your machine

### CODE REVIEW

**Review by rehman000 — APPROVED** [review]
Ok

**Comment by rehman000** [comment]
I reviewed it!

**Comment by jmuratov98** [comment]
Annotation 2020-04-15 132435

What is that supposed to be?

File: exit.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h> 
+
+int main(int argc, char* argv[]){
+
+    for(int i=0;i<4;i++){
+        char* argument = argv[i + 1];
+        printf("History Command is %s\n", argument);
+    }
```
Comment (jmuratov98) [inline exit.c:16]: This doesnt print the history?

File: exit.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    else{
+        wait(NULL);
+        printf("\nEnter 'Return key' back to main menu \n");
+        while(1)
+        {       
+            char input = getchar();
+            if(input == '\n')         
+                break;      
+        }
```
Comment (jmuratov98) [inline exit.c:39]: This wouldn't exit the shell.

**Review by jmuratov98 — CHANGES_REQUESTED** [review]

File: exit.c (hunk 1 above)
Comment (JiejayLan) [reply exit.c:16]: Should I get history from parent process?  [reference](https://stackoverflow.com/questions/18686114/cd-command-not-working-with-execvp)
Need to pass 4 arg manually
/exit  1 2 3 4

File: exit.c (hunk 2 above)
Comment (JiejayLan) [reply exit.c:39]: It works on my side. Not sure what happens.

**Review by jmuratov98 — APPROVED** [review]

**Pull request merged**